simplesat/sat/policy/undetermined_clause_policy.py

```diff
@@ -58,6 +58,11 @@ def _build_id_to_clauses(self, clauses):
         for c in clauses:
             for l in c.lits:
                 table[abs(l)].append(c)
+
+        # Make sure all installed packages appear in the table
+        for pid in self._installed_ids:
+            table[pid]
```

Comment:
We don't want to raise an exception; we want to make sure that the entry is always present. It's an edge case rather than an error. Another approach to this would be to use `fromkeys` and then set the default factory; I went with this because it was the suggestion here: https://bytes.com/topic/python/answers/803053-defaultdict-fromkeys-returns-surprising-defaultdict#post3194846